src/index.ts

```diff
@@ -588,14 +598,14 @@ async function setFeeOptions({
 				// for some chains, like arbitrum or base, providers tend to return an incorrect maxPriorityFeePerGas
 				// Sanity check so that it's never more than the base fee.
 				// exception: linea, where baseFee is hardcoded to 7 gwei (minimum allowed)
-				if (![59144, 59140].includes(chainId)) {
+				if (![59144, 59140].includes(_chainId)) {
 					if (BigInt(txOptions.maxPriorityFeePerGas) > lastBaseFeePerGas) {
 						txOptions.maxPriorityFeePerGas = lastBaseFeePerGas.toString()
 					}
 				}
 
 				// for polygon (137), set priority fee to min 40 gwei (they have a minimum of 30 for spam prevention)
-				if (chainId == 137) {
+				if (_chainId == 137) {
 					const minPriorityFee = ethers.utils.parseUnits('40', 'gwei')
 					if (ethers.BigNumber.from(txOptions.maxPriorityFeePerGas).lt(minPriorityFee)) {
 						txOptions.maxPriorityFeePerGas = minPriorityFee.toString()
```

Comment:
I think we're missing a 10% gaslimit default increase, no? Or does that behavior alrdy exist with gaslimit multiplier? It's hard to check on phone

Reply:
I'd like to test first without adding the 10% increase, we've used this function during the mantle campaign and there were no issues so not sure if we should change it before proper QA